lib/authz/permissions.go

```diff
@@ -1043,6 +1043,24 @@ func definitionForBuiltinRole(clusterName string, recConfig types.SessionRecordi
 						types.NewRule(types.KindClusterAuthPreference, services.RO()),
 						types.NewRule(types.KindRole, services.RO()),
 						types.NewRule(types.KindLock, services.RW()),
+						types.NewRule(types.KindAccessList, services.RW()),
```

Comment:
This rule gives blanket r/w access to access lists, and the one below uses a "where" clause. Don't these two conflict, is this one needed?

Reply:
Oversight on my part, I'll remove this line!